pkg/helmcatalog/actions/action.go

```diff
@@ -0,0 +1,11 @@
+package actions
+
+import (
+	"helm.sh/helm/v3/pkg/release"
+)
+
+type Action interface {
```

Comment:
I do not see that we are using this anywhere in the code atm, besides declaring a field in `HelmActions`, that remain even uninitialised. Please remove.

Reply:
Removed